core/include/kernel/tee_time.h

```diff
@@ -38,5 +38,7 @@ TEE_Result tee_time_get_ta_time(const TEE_UUID *uuid, TEE_Time *time);
 TEE_Result tee_time_get_ree_time(TEE_Time *time);
 TEE_Result tee_time_set_ta_time(const TEE_UUID *uuid, const TEE_Time *time);
 void tee_time_wait(uint32_t milliseconds_delay);
```

Comment:
+1 for the comment on `tee_time_wait()`, it's important to know it switches to Normal World.

I originally proposed `tee_time_mdelay()` because the Linux kernel has `mdelay()`, `udelay()` and `ndelay()` for busy wait loops. I also like `tee_time_busy_wait()` altough it won't allow us to later implement microsecond or nanosecond delays and keep a consistent naming.